Cargo.toml

```diff
@@ -6,6 +6,7 @@ members = [
     "bin/grpc-as",
     "bin/rvps-client",
 ]
+resolver = "2"
```

Comment:
i'm curious, why is it necessary? I read it's implied by setting `edition = 2021`, which might be less obscure. but fwiw I don't get any compilation errors when removing that line.

Reply:
I see this linter warning with 1.72 when running a basic `cargo build`. Some of our dependencies are explicitly setting `edition = 2021` which implies `resolver = 2` which is different from the the virtual workspace (we do use a virtual workspace) default resolver.

I feel like I'm rephrasing the warning itself, not sure this is giving a better explanation...